x/gov/interfaces.go

```diff
@@ -0,0 +1,42 @@
+package gov
+
+import (
+	weave "github.com/iov-one/weave"
+	"github.com/iov-one/weave/errors"
+	weavetest "github.com/iov-one/weave/weavetest"
+)
+
+// OptionLoader is needed to parse the raw_options data
+type OptionLoader func(raw []byte) (weave.Msg, error)
```

Comment:
As mention in the protobuf file, I think it would be better to call it `OptionDeserializer`. For me, it is more precise as _load_ feels very broad.
If you agree, this change will apply to other "loaders" as well.